skimage/segmentation/active_contour_model.py

```diff
@@ -48,6 +54,16 @@ def active_contour(image, snake, alpha=0.01, beta=0.1,
         Maximum iterations to optimize snake shape.
     convergence: float, optional
         Convergence criteria.
+    callback: function, optional
+        Called every 11 iterations with three args.
+        1st arg: (N, 2) ndarray -- the current snake.
+        2nd arg: int -- the total number of iterations completed.
+        3rd arg: float -- convergence criterion. This value is computed by

```

Comment:
We should be quite precise in describing how the convergence is computed.  Let's wait for your other research to figure out what exactly it should be.